nhais-adaptor/adaptor/outgoing/birth/tests/test_interchange_adaptor.py

```diff
@@ -2,14 +2,24 @@
 
 import adaptor.outgoing.birth.tests.fixtures as fixtures
 from testfixtures import compare
-import adaptor.outgoing.interchange_adaptor as adaptor
+
+from adaptor.fhir_helpers.fhir_creators import OperationName
+from adaptor.outgoing.birth.message_birth_adaptor import MessageBirthAdaptor
+from adaptor.outgoing.interchange_adaptor import InterchangeAdaptor
 
 
 class TestInterchangeAdaptor(unittest.TestCase):

```

Comment:
As discussed offline. We have decided to keep as is. Since it gives better coverage and more confidence that all the wiring for a new registration type is working as expected.